xbmc/video/VideoInfoTag.cpp

```diff
@@ -798,6 +798,18 @@ bool CVideoInfoTag::IsEmpty() const
           m_strPath.empty());
 }
 
+bool CVideoInfoTag::IsMediaInfoTag() const
```

Comment:
One way would be to use the `MediaTypes::IsCollection()` as a bookmark only makes sense on non-collection items.